Lib/idlelib/editor.py

```diff
@@ -414,6 +414,21 @@ def set_line_and_column(self, event=None):
 
 
     def createmenubar(self):
+        """Populate the menu bar widget for the editor window.
+
+        Each option on the menubar is itself a cascade-type Menu widget
```

Comment:
IMO in this case, the rest of the doc-string after the first line should be a comment instead, since it describes the method's code rather than it's purpose.